src/index.ts

```diff
@@ -21,6 +22,13 @@ export const VerifiedSnapStruct = object({
   id: string(),
   metadata: object({
     name: string(),
+    type: optional(enums(['account'])),
+    author: optional(string()),
+    website: optional(string()),
+    summary: optional(string()),
+    description: optional(string()),
+    audits: optional(array(string())),
+    tags: optional(array(string())),
```

Comment:
```suggestion
    tags: optional(array(string())),
    sourceCodeUrl: optional(string()),
    supportUrl: optional(string()),
```

Reply:
Should we keep `website` or apply the same pattern and rename it to `websiteUrl`?

Reply:
Could drop the `-url`?

Reply:
@Mrtenz only for `website` or for all fields (`sourceCode` and `support`)?

Reply:
All fields?

Reply:
I'm fine with nixing the url suffix

Reply:
All right, nixing the `Url` suffix after checking with @FrederikBolding on Slack :)